### PR TITLE
re-record docs

### PR DESCRIPTION
**Pull request by TomAugspurger**

### CODE REVIEW

File: gcsfs/tests/test_core.py [hunk 1]
```diff
@@ -667,15 +667,15 @@ def test_bigger_than_block_read(token_restore):
 
 @my_vcr.use_cassette(match=['all'])
 def test_current(token_restore):
-    from google.oauth2.credentials import Credentials
+    from google.auth import credentials
```
Comment (TomAugspurger) [inline gcsfs/tests/test_core.py:670]: I used a service account, not oauth.

Comment (martindurant) [reply gcsfs/tests/test_core.py:670]: Is there a base class that would include both possibilities?

Comment (martindurant) [reply gcsfs/tests/test_core.py:670]: (or is this already it...)

Comment (TomAugspurger) [reply gcsfs/tests/test_core.py:670]: This is it.

Comment (TomAugspurger) [reply gcsfs/tests/test_core.py:670]: It’s a little strange that they name them all Credentials.

**Pull request merged**